src/formio-complete.js

```diff
@@ -7,5 +8,7 @@ require('ng-file-upload');
 require('bootstrap');
 require('angular-ui-bootstrap');
 require('bootstrap-ui-datetime-picker/dist/datetime-picker');
+require('ckeditor/ckeditor');
```

Comment:
This needs to be removed. They should include the ckeditor manually within their app.